helm/istio/charts/tracing/templates/jaeger-all-in-one.yaml

```diff
@@ -12,7 +12,9 @@ spec:
   strategy: allInOne
 
   allInOne:
+    {{- if and .Values.jaeger.hub .Values.jaeger.tag }}
     image: {{ .Values.jaeger.hub }}/all-in-one:{{ .Values.jaeger.tag }}
```

Comment:
The image name should be jaeger-all-in-one-rhel7, @objectiser can you please confirm?

Reply:
I'm moving this to another issue